tests/test_csc.py

```diff
@@ -24,7 +24,7 @@
 
 class Harness:
     def __init__(self, config_dir=TEST_CONFIG_DIR):
-        salobj.test_utils.set_random_lsst_dds_domain()
+        salobj.set_random_lsst_dds_domain()
```

Comment:
This method is marked for deprecation. Please, use `salobj.set_random_lsst_dds_partition_prefix()` instead.